ui/redux/selectors/search.js

```diff
@@ -97,12 +105,15 @@ export const makeSelectWinningUriForQuery = (query: string) => {
 
   return createSelector(
     makeSelectClientSetting(SETTINGS.SHOW_MATURE),
+    makeSelectPendingClaimUrlForName(query),
     makeSelectClaimForUri(uriFromQuery),
     makeSelectClaimForUri(channelUriFromQuery),
-    (matureEnabled, claim1, claim2) => {
+    (matureEnabled, pendingClaimUrl, claim1, claim2) => {
       const claim1Mature = claim1 && isClaimNsfw(claim1);
       const claim2Mature = claim2 && isClaimNsfw(claim2);
-
+      if (pendingClaimUrl) {
+        return pendingClaimUrl;
```

Comment:
This should only return `pendingClaimUrl` if it's effective amount is greater than `claim1` and `claim2`

Reply:
refactored to return the claim. was simpler than I thought.